server/testutils/factories/cas1SpaceBookingSummary.ts

```diff
@@ -1,36 +1,63 @@
 import { Factory } from 'fishery'
 import { faker } from '@faker-js/faker'
-import {
-  Cas1SpaceBookingSummary,
-  Cas1SpaceBookingSummaryStatus,
-  Cas1SpaceCharacteristic,
-  PersonSummary,
-} from '@approved-premises/api'
+import { Cas1SpaceBookingSummary, Cas1SpaceCharacteristic, PersonSummary } from '@approved-premises/api'
 import { fullPersonSummaryFactory } from './person'
 import { DateFormats } from '../../utils/dateUtils'
 import cas1KeyworkerAllocationFactory from './cas1KeyworkerAllocation'
 import { roomCharacteristicMap } from '../../utils/characteristicsUtils'
 import cas1PremisesFactory from './cas1Premises'
+import { statusTextMap } from '../../utils/placements'
 
-const statuses: Array<Cas1SpaceBookingSummaryStatus> = [
-  'arrived',
-  'notArrived',
-  'arrivingWithin2Weeks',
-  'arrivingWithin6Weeks',
-  'departingWithin2Weeks',
-  'departed',
-  'arrivingToday',
-  'departingToday',
-  'overdueArrival',
-  'overdueDeparture',
-]
-const arrivedStatusses = ['arrived', 'departingWithin2Weeks', 'departed', 'departingToday', 'overdueDeparture']
-export default Factory.define<Cas1SpaceBookingSummary>(() => {
+const arrivedStatuses = ['arrived', 'departingWithin2Weeks', 'departed', 'departingToday', 'overdueDeparture']
+
+class Cas1SpaceBookingSummaryFactory extends Factory<Cas1SpaceBookingSummary> {
+  upcoming() {
+    return this.params({
+      actualArrivalDate: undefined,
+      actualDepartureDate: undefined,
+      isNonArrival: false,
+    })
+  }
+
+  current() {
+    const arrivalDate = faker.date.recent()
+    const departureDate = faker.date.soon()
```

Comment:
Might be better to give these some {days:} parameters as I think recent and soon both return dateTimes within 1 day by default.